src/devices/Dhtxx/samples/DhtSensor.sample.cs

```diff
@@ -15,26 +15,125 @@ internal class Program
         public static void Main(string[] args)
         {
             Console.WriteLine("Hello DHT!");
+            Console.WriteLine("Select the DHT sensor you want to use:");
+            Console.WriteLine(" 1. DHT10 on I2C");
+            Console.WriteLine(" 2. DHT11 on GPIO");
+            Console.WriteLine(" 3. DHT12 on GPIO");
+            Console.WriteLine(" 4. DHT21 on GPIO");
+            Console.WriteLine(" 5. DHT22 on GPIO");
+            var choice = Console.ReadKey();
+            Console.WriteLine();
+            if (choice.KeyChar == '1')
+            {
+                Console.WriteLine("Press any key to stop the reading");
+                Dht10II2c();
```

Comment:
Would be nice if this would be using the same code as below (except for the initialisation of course).

Reply:
Fully agree with you and at the same time, I was debating with myself. And I told myself that I would live it like this so it makes it easy for copy/paste. But I4m quite open. Happy to use the exact same code. Let's see other opinions.

Reply:
Debate with myself solved over night. Will just reuse. It's simple enough.